### ISSUE TITLE
BUG: Adding new calendar via web, downloading calendar on Gminder and no pull down item in quick add

### ISSUE THREAD

**Issue by noticz** [milowg/GMinder#7]
I add a new calendar, downloading it with Gminder, set the color up and then when going to quick add it is not listed in my quick add.

**Comment by noticz** [comment]
Here is a image to show you the problem
![gminder](https://cloud.githubusercontent.com/assets/11874318/7325502/d9b4ed00-ea7b-11e4-9c6c-35d6ccf20423.jpg)

**Comment by TekBear** [comment]
Does the problem persist after restarting GMinder?

**Comment by noticz** [comment]
Of course it does, even restarting the computer. It just has something to do with it not checking what new calendars there are.

**Comment by milowg** [comment]
I can't reproduce this. Can you give some detailed steps on how you created the calendar?

**Comment by noticz** [comment]
I just fixed it but I had to do an uninstall of the entire software and then delete anything left over. After installing it I now have everything back. I think the problem might have been from some files being left over from a different version causing problems. If anyone else has the same problem it can be fixed by...

Uninstall Gminder
Deleting the following folders
C:\Users\admin\AppData\Local\ReflectiveCode\GMinder.exe_Url*
C:\Users\admin\AppData\Roaming\GMinder.Auth.Store
C:\Users\admin\AppData\Roaming\GMinder
C:\Program Files\GMinder
Install Gminder again
